src/public/contact.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en-US">
+
+  <head>
+    <link rel="stylesheet" href="style.css">
+    <meta charset="utf-8">
+    <meta name="description" content="A blog about something awesome!">
+    <title>My Blog</title>
+  </head>
+
+
+  <body>
+    <header>
+      <h1>
+        Something Awesome!
+      </h1>
+    </header>
+    <div class="content-container">
+      <div class="nav-width">
+        <nav>
+          <ul class="no-bullets">
+            <li>
+              <a href="/about.html">About</a>
+            </li>
+            <li>
+              <a href="/contact.html">Contact</a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+
+      <div class="main-width">
+        <main>
+          <div id="contact-form" class="clearfix">
+            <h1>Contact Us!</h1>
+            <h2>
+              Please provide as much information as possible for us to help you with your inquiry :)
+            </h2>
+            <ul id="errors" class="">
+              <li id="info">
+                There were some problems with your form submission:
+              </li>
+            </ul>
+            <p id="success">
+              Thanks for your message! We will get back to you ASAP!
+            </p>
+            <form method="POST" action="/api/forms/contact">
+              <label for="name">Name: <span class="required">*</span></label>
+              <input type="text" id="name" name="name" value="" placeholder="Jimmy Choo" autofocus="autofocus" required/>
+
+              <label for="email">Email Address: <span class="required">*</span></label>
+              <input type="email" id="email" name="email" value="" placeholder="jchoo62@example.com" required/>
+
+              <label for="subject">Subject: <span class="required">*</span></label>
+              <input id="subject" name="subject"/>
+
+              <label for="message">Message: <span class="required">*</span></label>
```

Comment:
Looks like you got rid of the CSS for the `.required` class that makes the asterisk red. You should ether kill these spam tags with that class, or add some CSS to make use of them.